packages/rum/src/boot/rum.entry.ts

```diff
@@ -25,7 +26,7 @@ export interface RumUserConfiguration extends UserConfiguration {
   beforeSend?: (event: RumEvent) => void
 }
 
-export type RumGlobal = ReturnType<typeof makeRumGlobal>
+export type RumGlobal = ReturnType<typeof makeRumGlobal> & { addTiming?: AddTiming }
```

Comment:
What about not exposing it in RumGlobal type?
Since it will not necessarily be there

Reply:
if I don't include it on `RumGlobal` type, Typescript will complain whenever I try to access `addTiming`. I see two ways to fix this:
1- Cast `rumGlobal` to `any`.
2- Add a `InternalRumGlobal` type that contains `addTiming` and cast `rumGlobal` to that type when needed.
Which approach sounds good to you? is there a better way of handling this?

Reply:
> 1- Cast rumGlobal to any.

Could we do that only when we want to use addTiming?

```ts
(DD_RUM as any).addTiming(...)
```

Reply:
Yes, I will do that.